includes/REST_API/Stories_Autosaves_Controller.php

```diff
@@ -142,7 +142,7 @@ public function register_routes() {
 	 */
 	public function create_item( $request ) {
 		$kses = new KSES();
-		$kses->init();
+		$kses->add_filters();
 		$response = parent::create_item( $request );
 		$kses->remove_filters();
 		return $response;
```

Comment:
Same here. Since KSES now runs on `init`, no need to run it here too I think?

So we could just remove this entire method here.